internal/bundle/inspect/inspect.go

```diff
@@ -20,39 +20,68 @@ import (
 
 // Info represents information about a bundle.
 type Info struct {
-	Manifest    bundle.Manifest          `json:"manifest,omitempty"`
-	Signatures  bundle.SignaturesConfig  `json:"signatures_config,omitempty"`
-	WasmModules []map[string]interface{} `json:"wasm_modules,omitempty"`
-	Namespaces  map[string][]string      `json:"namespaces,omitempty"`
+	Manifest    bundle.Manifest            `json:"manifest,omitempty"`
+	Signatures  bundle.SignaturesConfig    `json:"signatures_config,omitempty"`
+	WasmModules []map[string]interface{}   `json:"wasm_modules,omitempty"`
+	Namespaces  map[string][]NamespaceInfo `json:"namespaces,omitempty"`
+	Annotations []*ast.AnnotationsRef      `json:"annotations,omitempty"`
 }
 
-func File(path string) (*Info, error) {
-	b, err := loader.NewFileLoader().WithSkipBundleVerification(true).AsBundle(path)
+type NamespaceInfo struct {
+	File          string
+	Title         string
+	Organizations []string
+}
+
+func File(path string, includeAnnotations bool, annotationsFilter []string) (*Info, error) {
+	b, err := loader.NewFileLoader().
+		WithSkipBundleVerification(true).
+		WithProcessAnnotation(true). // Always process annotations, for enriching namespace listing
+		AsBundle(path)
 	if err != nil {
 		return nil, err
 	}
 
 	bi := &Info{Manifest: b.Manifest}
 
-	namespaces := map[string][]string{}
+	namespaces := map[string][]NamespaceInfo{}
+	var modules []*ast.Module
 	for _, m := range b.Modules {
-		namespaces[m.Parsed.Package.Path.String()] = append(namespaces[m.Parsed.Package.Path.String()], filepath.Clean(m.Path))
+		ni := NamespaceInfo{
+			File: filepath.Clean(m.Path),
+		}
+		if pa := ast.FindPackageAnnotations(m.Parsed.Annotations); pa != nil {
+			// Regular annotations overriding not used for namespace info
+			ni.Title = pa.Title
+			ni.Organizations = pa.Organizations
```

Comment:
I'm tempted to not include the `organizations` table entry, as it could make the namespaces table grow to a quite large width. WDYT?